client/main.lua

```diff
@@ -325,10 +325,10 @@ end)
 
 local function enterVehicle(veh, indexgarage, type, garage)
     local plate = QBCore.Functions.GetPlate(veh)
-    if GetVehicleNumberOfPassengers(veh) == 0 then
+    if GetVehicleNumberOfPassengers(veh) ~= 0 then
```

Comment:
Are you supposed to be in your vehicle when you store it?
The else statement attached to this throws the vehicle occupied error.

Reply:
The enterVehicle function is checking if the vehicle I am using has passengers. If it has passengers, it cannot be stored. If it does not have passengers it checks that the vehicle is yours with the callback. Pretty simple if statement.

Yes, you have to be inside the vehicle to store it, since you need to be inside the zone with the vehicle.